release.yaml

```diff
@@ -2,3 +2,5 @@ longhorn:
   - 101.2.0+up1.4.0
 longhorn-crd:
   - 101.2.0+up1.4.0
+neuvector:
```

Comment:
if we’re doing this as an OOB release to modify an existing chart (which makes sense given the issue, but cc: @cbron in case that wasn’t the intention?), the release.yaml should only contain the one chart that is intended to be OOB released. all other charts should be removed in the first commit of this PR

Reply:
I thought this had to go straight to release branch because the file is autogenerated ? Or if you change old asset it would auto-generate correctly without the dot. But then you would have 2 different NV charts going out in the next release.

Reply:
@selvamt94 Can you add this fix to release.yaml file?